metpy/calc/kinematics.py

```diff
@@ -707,7 +707,9 @@ def lat_lon_grid_deltas(longitude, latitude, **kwargs):
         longitude, latitude = np.meshgrid(longitude, latitude)
 
     geod_args = {'ellps': 'sphere'}
-    geod_args.update(**kwargs)
+    if kwargs:
```

Comment:
Is this any more readable:
```python
geod_args = kwargs if kwargs else {'ellps': 'sphere'}
```
? I'm not sure if it is.

Reply:
I'd say it's 50/50. Fewer lines, but a more dense line.

Reply:
Life's too short to worry about crap like this. 😁